2

```diff
@@ -0,0 +1,52 @@
+<!-- <form accept-charset="UTF-8" method="POST" action="/users">
+  <label for="username">Username: </label>
```

Comment:
It's better to use the `users_path` helper instead of hardcoding the address` /users`. Thay way, if you customize the route, you won't' have to change it here.